testing/src/main/java/org/oppia/testing/TestCoroutineDispatcherRobolectricImpl.kt

```diff
@@ -43,6 +52,17 @@ class TestCoroutineDispatcherRobolectricImpl private constructor(
   private val totalTaskCount = AtomicInteger(0)
```

Comment:
There is one function in this file:
`createSortedTaskSet `
which gives warning that it needs API 24 and above.

<img width="491" alt="Screenshot 2020-09-02 at 11 42 05 PM" src="https://user-images.githubusercontent.com/9396084/92020640-160b8900-ed76-11ea-928d-618f1fdaf039.png">

Reply:
Yeah, this is slightly annoying--it's not actually correct. Robolectric is running on Java 8/9 based on the host, so this code should always work. It's not actually running in a real Android environment that is restricted by SDK levels.

Reply:
Great. Thanks for the explanation.